hathor/event/storage/event_storage.py

```diff
@@ -28,3 +28,7 @@ def save_event(self, event: BaseEvent) -> None:
     def get_event(self, key: int) -> Optional[BaseEvent]:
         """ Get a stored event by key"""
         raise NotImplementedError
+
+    @abstractmethod
+    def get_from_event(self, key: int, size: int) -> List[BaseEvent]:
+        """ Get a list of events from a key, with the size specified by the client"""
```

Comment:
Shouldn't we raise `NotImplementedError` here?

Reply:
This method does not exist anymore, but I'll make sure that all abstract methods raise `NotImplementedError`